src/test/scala/vct/newrewrite/FilterSpecIgnoreSpec.scala

```diff
@@ -0,0 +1,115 @@
+package vct.newrewrite
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import vct.col.ast._
+import vct.col.newrewrite.FilterSpecIgnore
+import vct.helper.{AstComparer, RewriteTestHelper, SimpleProgramGenerator}
+
+
+class FilterSpecIgnoreSpec extends AnyFlatSpec with Matchers {
+
+  //nothing should change
```

Comment:
If these comments are implemented, they can be removed right? Or do they still need to be done? IN that case they can be made TODO or can be converted into github issues?